examples/checkStealthAddress/index.ts

```diff
@@ -24,12 +24,12 @@ console.log(`View Tag: ${viewTag}`);
 
 // Check if an announcement (simulated here) is for the user
 const isForUser = checkStealthAddress({
-  ephemeralPublicKey, // From the announcement
-  schemeId: VALID_SCHEME_ID.SCHEME_ID_1,
-  spendingPublicKey: userSpendingPublicKey,
-  userStealthAddress: stealthAddress, // User's known stealth address
-  viewingPrivateKey: userViewingPrivateKey,
-  viewTag, // From the announcement
+	ephemeralPublicKey, // From the announcement
+	schemeId: VALID_SCHEME_ID.SCHEME_ID_1,
+	spendingPublicKey: userSpendingPublicKey,
+	userStealthAddress: stealthAddress, // User's known stealth address
```

Comment:
As discussed let's remove the formatting changes across the PR if they were not intended to be there.

Reply:
ran `bun run format` here; must have unintentionally been using biome or some stale format method.

https://github.com/ScopeLift/stealth-address-sdk/pull/64/commits/d5b2ae4243fdc992732a820c0accf509600f3d2d

Reply:
should be minimal formatting diffs now